Gruntfile.js

```diff
@@ -42,6 +42,59 @@ module.exports = function(grunt) {
 		grunt.task.run("qunit:each");
 	});
 
+	grunt.registerTask("browserstack", isBrowserStack ? function() {	
+		var eachfile = Array.prototype.slice.apply(arguments);
+		var taskList;
+		
+		if(eachfile.length) {
+			taskList = eachfile.map(function(v) {
+				return "browserstack_runner:" + v;
+			}, this);
+		} else {
+			var fs = require("fs");
+			
+			// fetch module name list from html files in test directory 
+			taskList = fs.readdirSync(
+				"test/"
+			).filter(function(val){
+				return val.split(".").length === 3 && val.split(".")[0] !== "buildMerge";
+			}).map(function(val){
+				return "browserstack_runner:" + val.split(".")[0];
+			});
+		}
+		grunt.task.run(taskList);
+	} : function() {
+		grunt.log.oklns(
+			"no BROWSER_STACK_USERNAME, BROWSER_STACK_ACCESS_KEY env"
+		);
+	});
+
+	grunt.registerTask("browserstack_runner", isBrowserStack ? function() {	
+		var fs = require("fs");
```

Comment:
https://github.com/naver/egjs/pull/97/files#diff-35b4a816e0441e6a375cd925af50752cR80 
https://github.com/naver/egjs/pull/97/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R38
Your indent is different from default indent. Could you check your indent?